lib/locale/languages.ts

```diff
@@ -2,545 +2,2612 @@ interface Language {
   code3: string
   code2: string
   name: string
+  flag?: string
 }
 
 export const LANGUAGES: Language[] = [
-  { code3: 'aar', code2: 'aa', name: 'Afar' },
-  { code3: 'abk', code2: 'ab', name: 'Abkhazian' },
-  { code3: 'ace', code2: '', name: 'Achinese' },
-  { code3: 'ach', code2: '', name: 'Acoli' },
-  { code3: 'ada', code2: '', name: 'Adangme' },
-  { code3: 'ady', code2: '', name: 'Adyghe; Adygei' },
-  { code3: 'afa', code2: '', name: 'Afro-Asiatic languages' },
-  { code3: 'afh', code2: '', name: 'Afrihili' },
-  { code3: 'afr', code2: 'af', name: 'Afrikaans' },
-  { code3: 'ain', code2: '', name: 'Ainu' },
-  { code3: 'aka', code2: 'ak', name: 'Akan' },
-  { code3: 'akk', code2: '', name: 'Akkadian' },
-  { code3: 'alb', code2: 'sq', name: 'Albanian' },
-  { code3: 'ale', code2: '', name: 'Aleut' },
-  { code3: 'alg', code2: '', name: 'Algonquian languages' },
-  { code3: 'alt', code2: '', name: 'Southern Altai' },
-  { code3: 'amh', code2: 'am', name: 'Amharic' },
-  { code3: 'ang', code2: '', name: 'English, Old (ca.450-1100)' },
-  { code3: 'anp ', code2: 'Angika', name: 'Angika' },
-  { code3: 'apa', code2: '', name: 'Apache languages' },
-  { code3: 'ara', code2: 'ar', name: 'Arabic' },
+  {
```

Comment:
nit: Wouldn't this be more readable using 1 line per item?

Reply:
Also, would 'this be more memory efficient by omitting `code2` if empty?

Reply:
prettier automatically formatted these but also, once compiled, I don't think it matters much, right?
cod3 is used for a different feature I think, can't recall exactly so we kinda need every item here.